.github/workflows/e2e-tests.yml

```diff
@@ -258,11 +223,25 @@ jobs:
         overwrite: true
 
 
-  playwright-electron:
+  electron:
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-14]
+        scripts: [".github/ci-cd-scripts/playwright-electron-linux.sh",".github/ci-cd-scripts/playwright-electron-windows.sh", ".github/ci-cd-scripts/playwright-electron-macos.sh"]
+        exclude:
+          - os: ubuntu-latest
+            scripts: ".github/ci-cd-scripts/playwright-electron-windows.sh"
+          - os: ubuntu-latest
+            scripts: ".github/ci-cd-scripts/playwright-electron-macos.sh"
+          - os: windows-latest
+            scripts: ".github/ci-cd-scripts/playwright-electron-linux.sh"
+          - os: windows-latest
+            scripts: ".github/ci-cd-scripts/playwright-electron-macos.sh"
+          - os: macos-14
+            scripts: ".github/ci-cd-scripts/playwright-electron-linux.sh"
+          - os: macos-14
+            scripts: ".github/ci-cd-scripts/playwright-electron-windows.sh"
```

Comment:
Clever workaround!

Reply:
Assuming my new fix works, I can remove this clever workaround.